Vagrantfile

```diff
@@ -19,14 +19,18 @@ end
 
 ####
 
+if ['up', 'provision', 'reload'].include?(ARGV[0])
+  unless system('ansible-playbook ansible/system_check/main.yml -i localhost,')
+    raise 'System check failed'
```

Comment:
I see in `ansible/system_check/main.yml` there is a fail.msg. However, this line seems to ignore this fail message for a generic System check failed message.

Is there a way to make the fail.msg more explicit when this issue occurs? I'm worried this issue could take users some time to debug because of this generic error message.

Note; the message _is_ hidden in the output of `vagrant provision`/`vagrant up`. Might just be a bit too hidden.

<img width="716" alt="screen shot 2018-05-10 at 8 54 56 am" src="https://user-images.githubusercontent.com/8646679/39870574-df322c86-542f-11e8-8b5e-a010effe0964.png">

Reply:
It is there in the first block of red — “Please update Ansible to version 2.4 by running `pip install --upgrade ansible.` As far as I know there is no way to pass the message from Ansible to Vagrant in this case — the best we could do is like what Vagrant does and suggest looking above for a more detailed message.